src/app/pages/Register/Register.module.css

```
@@ -0,0 +1,33 @@
+.container {
+  height: 100vh;
+  padding: 1.5em;
+  display: grid;
+  align-items: center;
+  grid-template-rows: auto;
+  row-gap: 1em;
+}
```

Would add a max-width (just for example 50%) so that the input field doesn´t increase over the whole page in full screen:
```suggestion
max-width: 50%;
margin: 0 auto;
}
```

---

Yes I do this! Only for example.